src/keygen/participant.rs

```diff
@@ -109,6 +109,10 @@ pub enum Status {
 /// 2021](https://eprint.iacr.org/2021/060.pdf). Figure 5.
 #[derive(Debug)]
 pub struct KeygenParticipant {
+    /// The current session identifier
+    sid: Identifier,
+    /// The current protocol input
+    input: (),
```

Comment:
suggestion: Same as broadcast, we don't actually have to keep this as a field because it's always going to be `()`.